Modules/Filtering/GPUAnisotropicSmoothing/itk-module.cmake

```diff
@@ -3,7 +3,7 @@ implement variations of anisotropic smoothing. This is an image denoising
 technique that strives for preserving edges on the images while smoothing regions
 of uniform intensity.  This type of filtering is convenient as a preprocessing
 stage of segmentation algorithms. You may find useful as well the filters in the
-\\\\ref ITKGPUSmoothingModule.")
+\\\\ref ITKGPUSmoothing.")
```

Comment:
Perhaps the `\\\\ref ITKGPUSmoothing module` ?

Reply:
ITK's scripts automatically add a `Module` prefix to these, so no need for that. Please have a look at the Doxygen files.